app/assets/stylesheets/navbar.scss

```diff
@@ -1,172 +1,145 @@
+    .nav-left {
+      gap: 1rem;
+      align-items: center;
 
-      a {
-        text-decoration: none;
-        color: $color-brand;
+      .nav-brand {
+        color: $color-primary-0;
+        font-size: 2rem;
+        padding: 1rem 0;
+        transition: color .25s;
+        line-height: 1;
       }
-    }
 
-    h2 {
-      font-weight: 200;
-      margin: 0 1rem;
-    }
-  }
-
-  .nav-right {
-    flex: 1;
-
-    .nav-link {
-      font-size: 3rem;
-      padding: 1.6rem 0 1.4rem;
-      margin: 0 0.75rem;
+      .nav-links {
+        gap: 0.5rem;
+    
+        @media screen and (max-width: $screen-lg) {
+          display: none;
+        }
 
-      .la-angle-down {
-        font-size: 1.5rem;
-        margin-left: 0.25rem;
-      }
+        .link {
+          padding: calc(1rem + 3px) 0.5rem 1rem;
+          border-bottom-color: transparent;
 
-      &.mobile-only {
-        display: none;
+          &:hover {
+            border-bottom-color: $color-complementary-0 !important;
+          }
+        }
       }
     }
 
-    .nav-dropdown {
-      .dropdown-button {
-        & i:first-child {
-          margin-left: 0.25rem;
+    .nav-right {
+      align-items: center;
+      height: fit-content;
+      padding: 0.75rem 0;
+      gap: 0.5rem;
+
+      .nav-authentication-btns {
+        @media screen and (max-width: $screen-lg) {
+          display: none;
         }
       }
 
-      .dropdown-content {
+      .nav-menu-expand {
         display: none;
-        position: absolute;
-        left: auto;
-        margin-top: 1.2rem;
-        background-color: $color-white;
-        box-shadow: 0 0.1rem 0.25rem $color-brand-light;
-        z-index: 1;
-
-        .nav-link {
-          display: block;
-          margin: 0 1rem;
-          color: $ink;
-          font-size: 1rem;
-
-          i {
-            font-size: 1.5rem;
-          }
 
-          &:hover {
-            color: $color-secondary;
-          }
+        @media screen and (max-width: $screen-lg) {
+          display: inline-block;
         }
       }
+    }
+  }
 
-      &:hover {
-        .dropdown-button {
-          color: $color-brand
-        }
+  .nav-links {
+    .link {
+      color: $color-primary-0;
+      font-weight: 600;
+      letter-spacing: -1px;
+      font-size: 1.2rem;
+      transition: border-bottom-color .25s;
+      border-bottom: 3px solid;
 
-        .dropdown-content {
-          display: block;
-        }
+      &:hover {
+        color: $color-primary-1;
       }
     }
+  }
 
-    .nav-mobile-expand {
-      display: none;
-      font-size: 1.5rem;
-    }
+  .nav-authentication-btns {
+    gap: 0.5rem;
+  }
 
+  .nav-menu-expand, .nav-menu-collapse {
+    font-size: 1.5rem;
+    line-height: 0;
+    padding: 0.6rem 1.5rem;
+    width: auto;
   }
-}
 
-@media screen and (max-width: $screen-md) {
-  .navbar {
-    .nav-left .nav-links {
-      display: none;
+  .nav-menu {
+    position: fixed;
+    right: 0;
+    top: 0;
+    z-index: 100;
```

Comment:
does this need to be 100? or can it be 10, or 2?

Reply:
I'm future proofing, based on previous experience you always need more Z's.

Reply:
haha. love it!